packages/wrangler/src/index.tsx

```diff
@@ -205,7 +205,9 @@ export async function main(argv: string[]): Promise<void> {
         try {
           await writeFile(
             destination,
-            `compatibility_date = "${compatibilityDate}"` + "\n"
+            `compatibility_date = "${compatibilityDate}"\nworkers_dev = true\nname = "${
+              args.name || path.basename(path.resolve(process.cwd()))
+            }"\n`
```

Comment:
remove `workers_dev`, and move `name` to the top of the output.
also I think now that this is multiple fields and could get complicated, we should use `TOML.stringify()` to generate this string.